src/lambda/commands/copyLambdaUrl.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultLambdaClient, LambdaClient } from '../../shared/clients/lambdaClient'
+import { LambdaFunctionNode } from '../explorer/lambdaFunctionNode'
+import globals from '../../shared/extensionGlobals'
+import { copyToClipboard } from '../../shared/utilities/messages'
+import { addCodiconToString } from '../../shared/utilities/textUtilities'
+
+export const NO_URL_INFO_MSG = 'No URL for Lambda function.'
+
+export async function copyLambdaUrl(
+    node: LambdaFunctionNode,
+    client: LambdaClient = new DefaultLambdaClient(node.regionCode),
+    copyToClipboardFunc: (data: string, label?: string) => Promise<void> = copyToClipboard
```

Comment:
Avoid explicit type annotations unless necessary
```suggestion
    copyToClipboardFunc = copyToClipboard
```